OpenRA.Mods.Common/Projectiles/Missile.cs

```diff
@@ -37,8 +37,8 @@ public class MissileInfo : IProjectileInfo
 		[Desc("Palette is a player palette BaseName")]
 		public readonly bool IsPlayerPalette = false;
 
-		[Desc("Should the projectile's shadow be rendered?")]
-		public readonly bool Shadow = false;
```

Comment:
Its likely that a (probably far) future PR will want to move the `ShadowColor` definitions to a global value as part of optimizing shadow rendering, so having special behaviour on `ShadowColor` for projectiles isn't a good idea.

I plan to go back to having this separate `Shadow` flag, which nicely also simplifies the update rule.